src/lwc/geFormRenderer/geFormRenderer.html

```diff
@@ -114,47 +101,114 @@
                             </c-util-illustration>
                         </div>
                     </template>
-                    <template if:false={isPermissionError}
-                              for:each={sections}
-                              for:item='section'>
-                        <c-ge-form-section key={section.id}
-                                           section={section}
-                                           onregisterpaymentwidget={handleRegisterPaymentWidget}
-                                           widget-config={widgetConfig}
-                                           onformfieldchange={handleFormFieldChange}
-                                           onformwidgetchange={handleFormWidgetChange}
-                                           form-state={formState}>
-                        </c-ge-form-section>
+                    <template if:true={isSingleGiftEntry}>
+                        <template if:true={showElevateTransactionWarning}>
```

Comment:
@sokosaurus It's weird that you can't see it, I re-added that block of mark-up here.

Reply:
Ok, I see that now too. Sorry for the confusion here! :)

Reply:
@schlossmj Look here

Reply:
Thanks for clarifying!